lms/djangoapps/bookmarks/signals.py

```diff
@@ -0,0 +1,13 @@
+from django.dispatch.dispatcher import receiver
+
+from xmodule.modulestore.django import SignalHandler
+
+
+@receiver(SignalHandler.course_published)
+def listen_for_course_publish(sender, course_key, **kwargs):  # pylint: disable=unused-argument
```

Comment:
I would name this based on its action, rather than it's condition (which is implied by the `@reciever` decorator).